src/lib/features/taskcluster_proxy.js

```diff
@@ -105,7 +106,7 @@ class TaskclusterProxy {
             task.runtime.log('Credentials updated', {
               taskId: task.status.taskId,
               runId: task.runId,
-              credentials: creds
+              credentials: _.omit(creds, ['accessToken']),
```

Comment:
it'd probably be good to omit the certificate, too.  Maybe just `clientId: creds.clientId`?

Reply:
Done.